tests/e2e/basic_test.go

```diff
@@ -156,9 +156,10 @@ func (s *BasicTestSuite) TestVmGroupVolumeIsolation(c *C) {
 	misc.LogTestStart(c.TestName())
 
 	// Initialize Config DB
-	admincli.ConfigInit(s.esx)
+	out, err := admincli.ConfigInit(s.esx)
+	c.Assert(err, IsNil, Commentf(out))
```

Comment:
I was thinking to extend verification further by invoking adminCLI config status and verify its output to find out configuration mode as changed or not that helps to reach out root cause easily later we will need such verification as well. 

IMO, we should little bit verification here to validate L#159 activity.